### PR TITLE
adding the first task

### PR DESCRIPTION
**Pull request by dohmeid**

### CODE REVIEW

File: index.html
Comment (ehabAmriahExalt) [inline index.html:1]: please rename the file to list.html

File: index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                    <option value="date">Date</option> 
+                    <option value="name">Name</option> 
+                </select>
+            </div>
+        </div>
+
+
+        <!--results section-->
+        <section class="results">
+            <div class="results-heading">
```
Comment (ehabAmriahExalt) [inline index.html]: please rename to "header"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                <h2>Results</h2>
+            </div>
+            
+            <div class="grid-container">
```
Comment (ehabAmriahExalt) [inline index.html]: please rename to "content"

File: index.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                <div class="square"></div>
```
Comment (ehabAmriahExalt) [inline index.html]: please rename to card

File: style.css [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+/*FOR THE HEADER (NAV BAR)*/
+header{
+    box-sizing: border-box;
+    background-color: #171515;
+    color: white;
+    flex: 80px 0 0; /*80px height, don't grow or shrink*/
+    padding: 1.3em 1.5em;
+    display: flex;
+    justify-content: space-between;
+}
```
Comment (ehabAmriahExalt) [inline style.css]: please add space between each 2 classes

File: style.css [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    overflow:auto;
+}
+.square {
+    background-color: lightgrey;
+}
+
+
+/******************************************************************/
+/*FOR THE FOOTER SECTION*/
+footer{
```
Comment (ehabAmriahExalt) [inline style.css]: please format the code

File: style.css
Comment (ehabAmriahExalt) [inline style.css:1]: - lets add the css files inside folder called "css"
- please rename this file to list.css

File: style.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+    width: 100%;
+}
+body{
+    background-color: white;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    height: 100%; 
+    width: 100%;
+    max-height: 100%;
```
Comment (ehabAmriahExalt) [inline style.css]: please remove this property if not needed

**Review by ehabAmriahExalt — CHANGES_REQUESTED** [review]
please check the comments

File: style.css (hunk 4 above)
Comment (dohmeid) [reply style.css]: to be honest I didn't really understand what do you mean by 'space', there is already a space-between the header elements

File: index.html
Comment (dohmeid) [reply index.html:1]: done

File: index.html (hunk 1 above)
Comment (dohmeid) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (dohmeid) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (dohmeid) [reply index.html]: done

File: style.css (hunk 4 above)
Comment (dohmeid) [reply style.css]: done

File: style.css (hunk 5 above)
Comment (dohmeid) [reply style.css]: done

File: style.css
Comment (dohmeid) [reply style.css:1]: done

File: style.css (hunk 6 above)
Comment (dohmeid) [reply style.css]: done

**Review by ehabAmriahExalt — APPROVED** [review]

**Pull request merged**